src/scenes/home/header/sideNav/sideNav.js

```diff
@@ -46,12 +46,10 @@ class SideNav extends Component {
       );
     }
     return (
-      <span>
-        <NavItem
-          to="/join" text="Join"
-          onClick={this.handleCloseClick}
-        />
-      </span>
+      <NavItem
+        to="/join" text="Join"
```

Comment:
Also, while you're in this file, make every attribute on a new line just for good standards.